cms/envs/test.py

```diff
@@ -142,7 +142,7 @@
 DISCUSSIONS_MICROFRONTEND_URL = "http://discussions-mfe"
 
 CACHES = {
-    # This is the cache used for most things. Askbot will not work without a
+    # This is the cache used for most things.will not work without a
     # functioning cache -- it relies on caching to load its settings in places.
```

Comment:
I think you should remove this whole sentence, so everything from "will" to "places."

Reply:
@dianakhuang I did it. Thanks